x-pack/plugins/security_solution/public/cases/components/create/form_context.test.tsx

```diff
@@ -0,0 +1,470 @@
+          ...sampleData,
+          connector: {
+            fields: {
+              impact: null,
+              severity: null,
+              urgency: null,
+            },
+            id: 'servicenow-1',
+            name: 'My Connector',
+            type: '.servicenow',
+          },
+        })
+      );
+    });
+
+    it('it should default to none if the default connector does not exist in connectors', async () => {
+      useCaseConfigureMock.mockImplementation(() => ({
+        ...useCaseConfigureResponse,
+        connector: {
+          id: 'not-exist',
+          name: 'SN',
+          type: ConnectorTypes.servicenow,
+          fields: null,
+        },
+        persistLoading: false,
+      }));
+
+      useConnectorsMock.mockReturnValue({
+        ...sampleConnectorData,
+        connectors: connectorsMock,
+      });
+
+      const wrapper = mount(
+        <TestProviders>
+          <FormContext onSuccess={onFormSubmitSuccess}>
+            <CreateCaseForm />
+            <SubmitCaseButton />
+          </FormContext>
+        </TestProviders>
+      );
+
+      await fillForm(wrapper);
+      wrapper.update();
+
+      await act(async () => {
+        wrapper.find(`[data-test-subj="create-case-submit"]`).first().simulate('click');
+      });
+
+      await waitFor(() => expect(postCase).toBeCalledWith(sampleData));
+    });
+  });
+
+  describe('Step 2 - Connector Fields', () => {
+    it(`it should submit a Jira connector`, async () => {
+      useConnectorsMock.mockReturnValue({
+        ...sampleConnectorData,
+        connectors: connectorsMock,
+      });
+
+      const wrapper = mount(
+        <TestProviders>
+          <FormContext onSuccess={onFormSubmitSuccess}>
+            <CreateCaseForm />
+            <SubmitCaseButton />
+          </FormContext>
+        </TestProviders>
+      );
+
+      await fillForm(wrapper);
+      await waitFor(() => {
+        expect(wrapper.find(`[data-test-subj="connector-settings-jira"]`).exists()).toBeFalsy();
+        wrapper.find('button[data-test-subj="dropdown-connectors"]').simulate('click');
+        wrapper.find(`button[data-test-subj="dropdown-connector-jira-1"]`).simulate('click');
+        wrapper.update();
+      });
+
+      await waitFor(() => {
+        wrapper.update();
+        expect(wrapper.find(`[data-test-subj="connector-settings-jira"]`).exists()).toBeTruthy();
+      });
+
+      act(() => {
+        wrapper
+          .find('select[data-test-subj="issueTypeSelect"]')
+          .first()
+          .simulate('change', {
+            target: { value: '10007' },
+          });
+      });
+
+      act(() => {
+        wrapper
+          .find('select[data-test-subj="prioritySelect"]')
+          .first()
+          .simulate('change', {
+            target: { value: '2' },
+          });
+      });
+
+      await act(async () => {
+        wrapper.find(`[data-test-subj="create-case-submit"]`).first().simulate('click');
+      });
```

Comment:
`act` can be removed here
```suggestion
      wrapper
        .find('select[data-test-subj="issueTypeSelect"]')
        .first()
        .simulate('change', {
          target: { value: '10007' },
        });
      
      wrapper
        .find('select[data-test-subj="prioritySelect"]')
        .first()
        .simulate('change', {
          target: { value: '2' },
        });

      wrapper.find(`[data-test-subj="create-case-submit"]`).first().simulate('click');
```